GenumKit/Stencil/Filters.swift

```diff
@@ -106,3 +126,83 @@ struct NumFilters {
     return "\(percent)%"
   }
 }
+
+struct ModelParsingFilters {
```

Comment:
`parameterList`
This one needs to create a list of params like: `x: String, y: Int`
The main problem here is that there is no trailing `,`. This means it's impossible to do in a template alone. So, I decided to use `join`! But `join` takes an array as a parameter, so I need to convert my input to an array. This is impossible to do in stencil. Basically, the main functionality which stencil seems to be missing is a map filter. Sadly, you can't just implement a `map` componenet, because component's are rendered. You need a `map` filter. But, since filters can't take parameters, you need to implement seperate filters for each `map`.